src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -102,32 +78,17 @@ private static String getTenant(String nsTenant) {
     return future;
   }
 
-  private static CQLWrapper initCqlWrapper(String query) throws FieldException {
-    CQL2PgJSON cql2pgJson;
-    cql2pgJson = new CQL2PgJSON(Collections.singletonList(REQUEST_TABLE + ".jsonb"));
-    return new CQLWrapper(cql2pgJson, query);
-  }
-
-  private static int countSucceeded(List<Future> futureList) {
-    int succeededCount = 0;
-    for (Future fut : futureList) {
-      if (fut.succeeded()) {
-        succeededCount++;
-      }
-    }
-    return succeededCount;
-  }
-
-
   /* For each expired request set Closed status and remove position*/
   private static List<Future> closeRequests(Vertx vertx, Context context, String tenant, List<Request> requestList) {
     List<Future> futureList = new ArrayList<>();
     if (!requestList.isEmpty()) {
       for (Request request : requestList) {
-        if (request.getStatus() == OPEN_NOT_YET_FILLED) {
-          request.setStatus(CLOSED_UNFILLED);
-        } else {
-          request.setStatus(CLOSED_PICKUP_EXPIRED);
+        switch (request.getStatus()) {
```

Comment:
I think Sonar is unhappy with this, and expects it be converted to two if statements

Reply:
fixed